### PR TITLE
added justrelease.rb

### PR DESCRIPTION
**Pull request by bilalyasar**

### CODE REVIEW

File: Library/Formula/justrelease.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Justrelease < Formula
+  homepage "github.com/justrelease"
+  url "https://github.com/justrelease/justrelease/releases/download/v1.0.0/justrelease-1.0.0.zip"
+  version "1.0.0"
```
Comment (bfontaine) [inline Library/Formula/justrelease.rb]: You can remove this line, the version should be detected from the URL.

File: Library/Formula/justrelease.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Justrelease < Formula
+  homepage "github.com/justrelease"
```
Comment (bfontaine) [inline Library/Formula/justrelease.rb]: Please add `https://` in front of this URL. Also please add a short description above this line using `desc "..."`.

File: Library/Formula/justrelease.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+       libexec.install "justrelease-#{version}.jar"
+       (bin/"justrelease").write <<-EOS.undent
+         #!/bin/sh
+         JUSTRELEASE_HOME="#{libexec}"
+         java -jar "$JUSTRELEASE_HOME/justrelease-#{version}.jar" $@
+       EOS
+  end
+
+  test do
+    system "false"
```
Comment (bfontaine) [inline Library/Formula/justrelease.rb]: Please add a test here.

**Comment by bilalyasar** [comment]
@bfontaine , thanks for the review, updated!

File: Library/Formula/justrelease.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+class Justrelease < Formula
+  desc "Install JustRelease Tool as CLI"
```
Comment (bfontaine) [inline Library/Formula/justrelease.rb]: It should be a description of Justrelease, not of the formula. You could use e.g. “CLI tool to release software libraries hosted on github.com”.

**Comment by bilalyasar** [comment]
@bfontaine , thanks again updated and squashed.

**Comment by bilalyasar** [comment]
@bfontaine , when this PR is merged? thanks.

**Comment by DomT4** [comment]
Our CI has been having some significant issues over the last week. It's slowly plowing through the backlog now. Please bear with it, Thanks!

**Comment by bilalyasar** [comment]
@bfontaine , @DomT4 what is the error? seems there is a failure.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by bilalyasar** [comment]
@DomT4 @bfontaine , is there any update?

**Comment by bfontaine** [comment]
mmh strange. @BrewTestBot test this please.

**Comment by bilalyasar** [comment]
@DomT4 @bfontaine, seems failed again,

**Comment by DomT4** [comment]
```
== /usr/local/Library/Formula/justrelease.rb ==
C:  9:  3: Use 2 (not 5) spaces for indentation.
C: 16:  4: Final newline missing.

1 file inspected, 2 offenses detected
Error: 4 problems in 1 formula

==> audit problems
justrelease:
 * File should end with a newline
 * A `test do` test block should be added
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
 * GitHub repository too new (<30 days old)
```

I'm afraid the last two errors are "fatal" in terms of Homebrew submissions. We require those minimum standards to ensure software gets maintained over time and has some form of popularity. I'm afraid until it meets both those requirements, we're going to have to pass on this.

Once those requirements are satisfied, we welcome a resubmission to Homebrew. Thank you for being willing to contribute.

**Comment by DomT4** [comment]
Side note: You should completely feel free to maintain it in a tap until it does meet those requirements. There's some documentation on that [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md).

**Pull request closed without merging**